### PR TITLE
Prep

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

**Comment by amaseda** [comment]
:+1: PR looks good. Will dive into the new Tunr repo once I'm done with some grading.

File: readme.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Alright! Let's get started with some code!
 
 ### Setup SQL - WDI (I Do - 5 / 40)
 > Throughout the day, I'll be doing some code simulating a "wdi application" then you will code along with our Tunr
-applications.
+applications. Please **do not** code along the wdi application.
```
Comment (nolds9) [inline readme.md:73]: :+1:

File: readme.md [hunk 2]
```diff
@@ -166,18 +166,6 @@ gem "pry"  # this gem allows access to REPL
...
-If you did not complete the initial SQL setup for Tunr above:
- - In your cloned copy, run `git fetch https://github.com/ga-dc/tunr_sinatra` to get the most recent branches
- - Please `git checkout` to the `2_active_record_starter` branch
- - Then you will need to create the db with `createdb tunr_db`
- - Then load the schema file with `psql -d tunr_db < db/schema.sql`
- - Next run the seed file with `psql -d tunr_db < db/seed.sql`
- - Follow the steps from `Part 2.1` in the link below
-
-[Part 2.1 - Create the Artist Model Using Active Record](https://github.com/ga-dc/tunr_sinatra/tree/2_active_record_starter#part-21---create-the-artist-model-using-active-record)
-
```
Comment (nolds9) [inline readme.md:179]: Much better to move this to the initial setup for [tunr](https://github.com/ga-wdi-exercises/tunr-active-record#part-11---create-the-artist-model-using-active-record). Do they already do the first part in SQL class, or do you need to intro it here?

**Comment by nolds9** [comment]
Think you just need to remove the word `Opening` from the framing to make tests pass

File: readme.md (hunk 2 above)
Comment (joe-gz) [inline readme.md:179]: note, we don't touch tunr in SQL/Domain modeling class. The example and exercises are all based on a library

File: readme.md [hunk 3]
```diff
@@ -133,9 +133,9 @@ It'll be nice going forward with your application that we package the schema up
 
 **NOTE:** If you already ran your Tunr schema in the Domain Modeling / SQL class, you do not need to complete this portion.  
 
-[Part 1 - Database/Schema](https://github.com/ga-dc/tunr_sinatra/tree/2_active_record_starter#part-1---database--schema)
+[solution code](https://github.com/ga-wdi-exercises/tunr-active-record/archive/v1.0.zip)
```
Comment (nolds9) [inline readme.md:136]: are you planning on having them clone down any [starter](https://github.com/ga-wdi-exercises/tunr-active-record#local-setup)?

File: readme.md (hunk 2 above)
Comment (jshawl) [inline readme.md:179]: :+1: will reframe - thanks for heads up!

File: readme.md (hunk 3 above)
Comment (jshawl) [inline readme.md:136]: na, just all from scratch. there are downloadable zip files if someone is not caught up

**Pull request merged**